servers/rendering/renderer_rd/effects/sort_effects.h

```diff
@@ -31,6 +31,7 @@
 #ifndef SORT_EFFECTS_RD_H
 #define SORT_EFFECTS_RD_H
 
+#include "servers/rendering/renderer_rd/shader_rd.h"
```

Comment:
Quick fix due to [build failing](https://github.com/godotengine/godot/actions/runs/12693042092/job/35379786940?pr=101352) because of erroneous cache restores.

I have [previously established](https://github.com/godotengine/godot/pull/100293#issuecomment-2556731863) that including a non-generated file explicitly solves such cache issues for good.